app/src/androidTest/java/org/simple/clinic/PatientFaker.kt

```diff
@@ -9,10 +10,10 @@ import org.simple.clinic.patient.sync.PatientPayload
 import org.simple.clinic.patient.sync.PatientPhoneNumberPayload
 import org.threeten.bp.Instant
 import java.util.UUID
+import javax.inject.Inject
 
-object PatientFaker {
-
-  private val faker = Faker()
+@AppScope
+class PatientFaker @Inject constructor(private val faker: Faker) {
```

Comment:
Are we making `PatientFaker` a part of our DI graph because it now depends on `Faker`?

Reply:
Yes.